mod/cornercutter/Enum/ConfigOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace cornercutter.Enum
+{
+    // This is a bitmask for all the mod specific config options (i.e, enabled per mod).
+    [Flags]
+    public enum ConfigOptions
+    {
+        NoneSelected = 0,
+        ConfigPerFloor = 1 << 0,

```

Comment:
Sometimes you use `None` and sometimes you use `NoneSelected`. Maybe try and be consistent across the enums